docs/data-sources/application_epg.md

```diff
@@ -127,7 +127,6 @@ data "aci_application_epg" "example_application_profile" {
   * `switching_mode` (switchingMode) - (string) The switching mode of the Relation To Domain object.
   * `target_dn` (tDn) - (string) The distinguished name of the target Domain object.
   * `untagged` (untagged) - (string) The untagged status of the Relation To Domain object.
-  * `vnet_only` (vnetOnly) - (string) The VNET only status of the Relation To Domain object.
```

Comment:
The version associated with this is in the meta has a multiple range format which looks like this "versions": "4.2(6d)-4.2(7w),5.1(2e)-"
We decided to exclude this attribute for now because TF aci has not yet been released and until we come up with an appropriate function to handle this for comparison..